internal/rest/avi_obj_pool.go

```diff
@@ -364,7 +364,7 @@ func (rest *RestOperations) AviPoolCacheAdd(rest_op *utils.RestOp, vsKey avicach
 								statusOption.ObjType = lib.MultiClusterIngress
 							}
 							utils.AviLog.Debugf("key: %s Publishing to status queue, options: %v", updateOptions.ServiceMetadata.IngressName, utils.Stringify(statusOption))
```

Comment:
we should also update the logs to match the changes at here and all relevant places